wordpress-updatable/Makefile

```diff
@@ -0,0 +1,136 @@
+TAG ?= 0.1
```

Comment:
nit: What about 
wordpress-managed-update/

It is easier to correlate with the actual name of the feature.

Reply:
`managed-updates` is long; `updatable` is actually a more descriptive name for the solution.